base-image/Dockerfile

```diff
@@ -121,7 +121,8 @@ ONBUILD RUN echo "Checking for pip 'requirements.txt'..." \
         ; [ -d binder ] && cd binder \
         ; [ -d .binder ] && cd .binder \
         ; if test -f "requirements.txt" ; then \
-        ${NB_PYTHON_PREFIX}/bin/pip install --no-cache-dir -r requirements.txt \
+        export PATH=${NB_PYTHON_PREFIX}/bin:${PATH} \
```

Comment:
@jhamman can you just add this to the ENV block so that it's in one place? https://github.com/pangeo-data/pangeo-docker-images/blob/690362786bc1da03e2cb953982a14d64a91815b4/base-image/Dockerfile#L18